src/org/zaproxy/zap/extension/pscanrulesAlpha/ImageLocationScanner.java

```diff
@@ -86,8 +75,12 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		if (logger.isDebugEnabled()) {
 			start = System.currentTimeMillis();
 		}
-				
-        String CT = msg.getResponseHeader().getHeader("Content-Type");
+		
+        // Mnemonic: CT ==> Content-Type
+        String CT = msg.getResponseHeader().getHeader("Content-Type").toLowerCase();
```

Comment:
The ```toLowerCase``` should be done after checking that the ```CT``` is not ```null```.
Adding comment again as I added it in the commit instead of PR, sorry.